Makefile

```diff
@@ -101,7 +103,7 @@ clone: # Clone REANA source code repositories locally.
 build: # Build REANA client and cluster components.
 	@echo -e "\033[1;32m[$$(date +%Y-%m-%dT%H:%M:%S)]\033[1;33m reana:\033[0m\033[1m make build\033[0m"
 	source ${HOME}/.virtualenvs/${VENV_NAME}/bin/activate && \
-	eval $$(minikube docker-env --profile ${MINIKUBE_PROFILE})  && \
+	minikube docker-env --profile ${MINIKUBE_PROFILE} 2&>1 /dev/null && eval $$(minikube docker-env --profile ${MINIKUBE_PROFILE}) && \
```

Comment:
This creates files `1` in the current working directory, I'll push a quick fix before merging...

Reply:
... oops, I don't have rights to push onto your feature branch.  Please fix simply by using ` ... > /dev/null` instead of `... 2&>1 /dev/null` and repush yourself.